assets/js/pluginsidebar/components/sidebar/index.js

```diff
@@ -41,6 +46,13 @@ const {
 class Sidebar extends React.PureComponent {
   // Define PropTypes for this component.
   static propTypes = {
+    appleNewsNotices: PropTypes.arrayOf(PropTypes.shape({
+      dismissed: PropTypes.bool,
+      dismissible: PropTypes.bool,
+      message: PropTypes.string,
+      type: PropTypes.string,
```

Comment:
```suggestion
      timestamp: PropTypes.number,
      type: PropTypes.string,
```

Reply:
Great point - I'll need to add that to a few places where I'm manually creating notifications, also. Thanks for catching!